OasisDEX/oasis-borrow
# First PR for generalise AAVE

- consumes the latest version of the library
- adds tokens to context:  `collateral`, `debt` and `deposit`
- fixes  a bunch of things in the sidebar
- sorts the top-two values in the simulate section
- loads the position using the library
- create, adjust close ETH/USDC

Todo (functional):

- the rest of the values in the simulate section
- make the simulate section more generic for a collateral token

Todo (technical):

- tidy reading position from lib (move from app context to machines)

Testing:
- create adjust close AAVE Earn, and ensure values correct
- create adjust close AAVE Multiply
